common/serializer.h

```diff
@@ -264,6 +264,29 @@ class Serializer {
 		}
 	}
 
+	/**
+	 * Sync a U32-string
+	 */
+	void syncString32(U32String &str, Version minVersion = 0, Version maxVersion = kLastVersion) {
+		if (_version < minVersion || _version > maxVersion)
+			return; // Ignore anything which is not supposed to be present in this save game version
+
+		uint32 len = str.size();
+
+		syncAsUint32LE(len);
+
+		if (isLoading()) {
+			uint32 *sl = new uint32[len];
```

Comment:
Maybe use `U32String::value_type` rather than `uint32` in case we decide to use something else (such as `char32_t`).

Reply:
done